src/pages/workspace/Settings.vue

```diff
@@ -1,11 +1,200 @@
+          </Paragraph>
+          <Button small @click="submitProfile">submit</Button>
+        </div>
+      </Card>
+      <Card>
+        <div>
+          <Header quaternary>Workspace</Header>
+          <Paragraph>name: {{ workspace ? workspace.name : '' }}</Paragraph>
+          <Paragraph>
+            created:
+            {{ (workspace ? workspace.created : null) || '?' }}
+          </Paragraph>
+          <Paragraph>Members:</Paragraph>
+          <ul v-if="workspace && workspace.members">
+            <li
+              v-for="m in workspace.members"
+              :key="workspace.members.indexOf(m)"
+            >
+              {{ `${m.callname} (${m.email})` }}
+            </li>
+          </ul>
+          <InviteForm @error="onError"></InviteForm>
+        </div>
+      </Card>
+      <Card class="danger-zone">
+        <div>
+          <Header quaternary>Danger Zone</Header>
+          <Button @click="confirmDelete = true">delete this workspace</Button>
+          <Confirmation
+            v-if="confirmDelete"
+            title="Are you sure?"
+            description="Do you really want to delete this workspace? This action cannot not be undone! Please type the name of this workspace if you want to proceed."
+            :confirmation="workspace.name"
+            @confirmed="deleteWorkspace"
+          ></Confirmation>
+        </div>
+      </Card>
+    </div>
   </div>
 </template>
 
 <script>
-export default {
+import gql from 'graphql-tag'
+import Component from '/components/sharedScripts/component'
+import LabeledInput from '/components/molecular/LabeledInput'
+import Confirmation from '/components/molecular/Confirmation'
+import InviteForm from './settings/InviteForm'
+
+export default new Component({
   name: 'Settings',
-}
+  apollo: {
+    user: gql`
+      {
+        user {
+          name
+          callname
+          email
+          id
+        }
+      }
+    `,
+    workspaces: {
+      query: gql`
+        {
+          workspaces {
+            name
+            created
+          }
+        }
+      `,
+      client: 'work',
+    },
+    workspace: {
+      query: gql`
+        query getWorkspace($name: String!) {
+          workspace(name: $name) {
+            name
+            created
+            members {
+              callname
+              email
+            }
+          }
+        }
+      `,
+      client: 'work',
+      variables() {
+        return {
+          name: localStorage.getItem('workspace'),
+        }
+      },
+    },
+  },
+  components: {
+    LabeledInput,
+    InviteForm,
+    Confirmation,
+  },
+  data: {
+    user: {
+      name: '',
+      callName: '',
+      email: '',
+    },
+    workspaces: [],
+    confirmDelete: false,
+  },
+  methods: {
+    submitProfile: function() {
+      this.$apollo.mutate({
+        mutation: gql`
+          mutation { 
+            setName(name: "${this.user.name}" callname: "${
+          this.user.callname
+        }") {
```

Comment:
Please use GraphQL variables here, and maybe re-do the formatting :)

Reply:
done 7808796